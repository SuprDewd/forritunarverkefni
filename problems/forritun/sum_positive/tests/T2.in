-173
563
-636
-392
413
-127
-360
-273
351
-606
