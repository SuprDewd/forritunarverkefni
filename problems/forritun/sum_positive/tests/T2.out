1327
3
7
